File: python_backend/game_logic/ranges/specialMoves/castle/get_rook_locations.py
from game_logic.ranges.specialMoves.castle.rook_locations import rook_locations


def get_rook_locations(king_loc):
    """using king_loc to get starting rook locations"""
    return rook_locations[king_loc]


if __name__ == "__main__":

    print("(5, 1)")
    print(get_rook_locations((5, 1)))
    print("(5, 8)")
    print(get_rook_locations((5, 8)))

File: node.js/ranges/specialMoves/castle/getRookLocations.js
var  rookLocations  = require("./rookLocations");


function getRookLocations(kingLoc) {
    /*using kingLoc to get starting rook locations**/
    return rookLocations[kingLoc]
}

module.exports = getRookLocations;

if (require.main === module) {
    console.log("[5, 1]")
    console.log(getRookLocations([5, 1]))
    console.log("[5, 8]")
    console.log(getRookLocations([5, 8]))
}

